openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -38,11 +37,10 @@
     _check_user_auth_flow
 )
 from openedx.core.djangolib.testing.utils import CacheIsolationTestCase, skip_unless_lms
-from openedx.core.djangoapps.site_configuration.tests.mixins import SiteMixin
 from openedx.core.lib.api.test_utils import ApiTestCase
 from openedx.features.enterprise_support.tests.factories import EnterpriseCustomerUserFactory
-from common.djangoapps.student.models import LoginFailures
-from common.djangoapps.util.password_policy_validators import DEFAULT_MAX_PASSWORD_LENGTH
+from openedx_events.tests.utils import OpenEdxEventsTestMixin  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
Why do we want this imported at this line in particular?

Reply:
I used a PyCharm "Optimize Imports" tool action, which, among other things, arranges imports alphabetically. It would appear that the underscore in "openedx_events" places this import in lower alphabetic order relative to the dot in "openedex.features"